### PR TITLE
[TestbedV2] Migrate multi-asic test jobs to TestbedV2

### PR DESCRIPTION
**Pull request by yutongzhang-microsoft**
### Description of PR

Migrate multi-asic test jobs to TestbedV2

Summary:
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 201911
- [ ] 202012
- [ ] 202205

### CODE REVIEW

**Comment by wangxin** [comment]
Can you also help fix the pre-commit issues? It seems that there are whitespaces in the end of some lines in `.azure-pipelines/run-test-scheduler-template.yml`.
You can configure your IDE/Editor to automatically trim whitespace in the end of line.

**Review by yejianquan — APPROVED** [review]
LGTM

File: .azure-pipelines/pr_test_scripts.yaml [hunk 1]
```diff
@@ -107,13 +107,9 @@ multi-asic-t1-lag:
   - tacacs/test_accounting.py
   - tacacs/test_authorization.py
   - tacacs/test_jit_user.py
-  - tacacs/test_ro_disk.py
```
[the remark this answers is not in the document]
Comment (yutongzhang-microsoft) [reply .azure-pipelines/pr_test_scripts.yaml:110]: This case is not running in multi-asic

File: .azure-pipelines/pr_test_scripts.yaml [hunk 2]
```diff
@@ -107,13 +107,9 @@ multi-asic-t1-lag:
...
   - tacacs/test_ro_user.py
   - tacacs/test_rw_user.py
 
-multi-asic-t1-lag-pr:
```
[the remark this answers is not in the document]
Comment (yutongzhang-microsoft) [reply .azure-pipelines/pr_test_scripts.yaml:114]: This part is no use.

**Pull request merged**